sessions/forecasting-models.qmd

```diff
@@ -330,7 +354,9 @@ What do you notice about the forecasts from the more mechanistic and more statis
 ::: {.callout-note collapse="true"}
 ## Solution
 - The more mechanistic model captures the downturn in the data very well.
```

Comment:
This doesn't quite make sense to me as we don't have a downturn in the data at this point, since this is cutoff before the peak? Is this leftover from before we started using the longer outbreak dataset?